### PR TITLE
Update sdk-js references

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
#### Description of the pull request (what is new / what has changed)
updates from `erdjs` to `sdk-js` + added redirects

#### Did you test the changes locally?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/developers/creating-wallets.md [hunk 1]
```diff
@@ -9,52 +9,44 @@ Although wallets are commonly created through the [MultiversX Web Wallet](https:
...
-  accountIndex.toString(),
-  ""
-);
-let privateKey = Buffer.from(privateKeyHex, "hex");
-let keyFileObject = account.generateKeyFileFromPrivateKey(privateKey, password);
-let keyFileJson = JSON.stringify(keyFileObject, null, 4);
-
-fs.writeFileSync("myKeyFile.json", keyFileJson);
+const mnemonic = Mnemonic.generate();
+const password = "my non secure password";
```
Comment (andreibancioiu) [inline docs/developers/creating-wallets.md:42]: Perhaps `"insert a password here"`? (opinion).

File: docs/developers/signing-transactions/tools-for-signing.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ In either case, the output file looks like this:
 }
 ```
 
-## **Sign using [erdwalletjs-cli](/sdk-and-tools/erdwalletjs-cli)**
+## **Sign using [sdk-js-wallet-cli](/sdk-and-tools/sdk-js-wallet-cli)**
```
Comment (andreibancioiu) [inline docs/developers/signing-transactions/tools-for-signing.md:50]: The command alias below will change (can be left for future PR).

File: docs/sdk-and-tools/sdk-js/sdk-js.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+| [sdk-web-wallet-provider](https://www.npmjs.com/package/@multiversx/sdk-web-wallet-provider)     | [Github](https://github.com/multiversx/mx-sdk-js-web-wallet-provider)     | Sign using the MultiversX web wallet.                                                         |
+| [sdk-wallet-connect-provider](https://www.npmjs.com/package/@multiversx/sdk-web-wallet-provider) | [Github](https://github.com/multiversx/mx-sdk-js-wallet-connect-provider) | Sign using Wallet Connect.                                                                    |
+| [sdk-extension-provider](https://www.npmjs.com/package/@multiversx/sdk-extension-provider)       | [Github](https://github.com/multiversx/mx-sdk-js-extension-provider)      | Sign using the Maiar DeFi Wallet (browser extension).                                         |
+
+:::important
+For all purposes, **we recommend using [dapp-core](/sdk-and-tools/dapp-core)** instead of integrating the signing providers on your own.
```
Comment (andreibancioiu) [inline docs/sdk-and-tools/sdk-js/sdk-js.md:29]: Dapp core is now sdk-dapp.

File: docusaurus.config.js [hunk 4]
```diff
@@ -186,20 +186,32 @@ const config = {
...
-        //   if (existingPath.includes("/community")) {
-        //     // Redirect from /docs/team/X to /community/X and /docs/support/X to /community/X
-        //     return [
-        //       existingPath.replace("/community", "/docs/team"),
-        //       existingPath.replace("/community", "/docs/support"),
-        //     ];
-        //   }
-        //   return undefined; // Return a falsy value: no redirect created
-        // },
+        createRedirects(existingPath) {
```
Comment (andreibancioiu) [inline docusaurus.config.js:201]: :+1:

**Review by andreibancioiu — DISMISSED** [review]

File: docs/developers/creating-wallets.md [hunk 5]
```diff
@@ -9,52 +9,44 @@ Although wallets are commonly created through the [MultiversX Web Wallet](https:
 
 ## **Generate a new mnemonic**
 
-Using [erdwalletjs-cli](https://www.npmjs.com/package/@elrondnetwork/erdwalletjs-cli), a mnemonic phrase (24 words) can be generated as follows:
+Using [erdwalletjs-cli](https://www.npmjs.com/package/@multiversx/sdk-wallet-cli), a mnemonic phrase (24 words) can be generated as follows:
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/developers/creating-wallets.md]: done

File: docs/developers/creating-wallets.md [hunk 6]
```diff
@@ -9,52 +9,44 @@ Although wallets are commonly created through the [MultiversX Web Wallet](https:
...
-console.log(mnemonic);
+let mnemonic = Mnemonic.generate();
+let words = mnemonic.getWords();
+console.log(words);
 ```
 
 ## **Deriving a JSON key-file (from mnemonic)**
 
-Using [erdwalletjs-cli](https://www.npmjs.com/package/@elrondnetwork/erdwalletjs-cli), a JSON key-file can be obtained as follows:
+Using [sdk-wallet-cli](https://www.npmjs.com/package/@multiversx/sdk-wallet-cli), a JSON key-file can be obtained as follows:
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/developers/creating-wallets.md:30]: done

File: docs/developers/creating-wallets.md (hunk 1 above)
Comment (bogdan-rosianu) [reply docs/developers/creating-wallets.md:42]: done

File: docs/developers/signing-transactions/tools-for-signing.md (hunk 2 above)
Comment (bogdan-rosianu) [reply docs/developers/signing-transactions/tools-for-signing.md:50]: changed

File: docs/sdk-and-tools/sdk-js/sdk-js.md (hunk 3 above)
Comment (bogdan-rosianu) [reply docs/sdk-and-tools/sdk-js/sdk-js.md]: done

**Review by radumojic — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**